docs/src/main/asciidoc/configuration.adoc

```diff
@@ -1,86 +1,87 @@
+|spring.cloud.azure.*_<azure-service>_.client*
 |To configure the transport clients underneath one Azure service SDK. 
 
-|*spring.cloud.azure.<azure-service>*.credential
+|spring.cloud.azure.*_<azure-service>_.credential*
 |To configure how to authenticate with Azure Active Directory for one Azure service SDK.
 
-|*spring.cloud.azure.<azure-service>*.profile
+|spring.cloud.azure.*_<azure-service>_.profile*
 |To configure the Azure cloud environment for one Azure service SDK. 
 
-|*spring.cloud.azure.<azure-service>*.proxy
+|spring.cloud.azure.*_<azure-service>_.proxy*
 |To configure the proxy options for one Azure service SDK. 
 
-|*spring.cloud.azure.<azure-service>*.retry
-|To configure the retry options apply to one Azure service SDK. 
-|===
+|spring.cloud.azure.*_<azure-service>_.retry*
+|To configure the retry options apply to one Azure service SDK. The retry options has supported part of the SDKs, there's no `spring.cloud.azure.cosmos.retry`.
+|=== 
+
 
-There are some properties that could be shared among different Azure services, for example using the same service principal to access Azure Cosmos DB and Azure Event Hubs. Spring Cloud Azure 4.0 allows developers to define properties that apply to all Azure SDKs in the namespace `spring.cloud.azure`.
+There are some properties that could be shared among different Azure services, for example using the same service principal to access Azure Cosmos DB and Azure Event Hubs. Spring Cloud Azure 4.0 allows developers to define properties that apply to all Azure SDKs with the prefix `spring.cloud.azure`.
 
 .Global properties
 [cols="1,3", options="header"]
 |===
 |Property  |Description
 
-|*spring.cloud.azure*.client
+|spring.cloud.azure.*client*
 |To configure the transport clients apply to all Azure SDKs by default. 
 
-|*spring.cloud.azure*.credential
+|spring.cloud.azure.*credential*
 |To configure how to authenticate with Azure Active Directory for all Azure SDKs by default. 
 
-|*spring.cloud.azure*.profile
+|spring.cloud.azure.*profile*
 |To configure the Azure cloud environment for all Azure SDKs by default. 
 
-|*spring.cloud.azure*.proxy
+|spring.cloud.azure.*proxy*
 |To configure the proxy options apply to all Azure SDK clients by default. 
 
-|*spring.cloud.azure*.retry
+|spring.cloud.azure.*retry*
 |To configure the retry options apply to all Azure SDK clients by default. 
 |===
 
 
 NOTE: Properties configured under each Azure service will override the global configurations.
 
-The configuration properties' prefixes have been unified to `spring.cloud.azure` namespace since Spring Cloud Azure 4.0, it will make configuration properties more consistent and more intuitive. Here's a quick review of the prefixes of supported Azure services.
+Spring Cloud Azure unifies configuration properties' prefixes to `spring.cloud.azure` since 4.0, which will make configuration properties more consistent and more intuitive. Here's a quick review of the prefixes of each supported Azure service.
```

Comment:
Here's a quick review of the prefixes of each supported Azure service.
to be---->
Here's a quick review of each prefix for supported Azure services.

Reply:
will change to `Here's a quick review of prefixes for supported Azure services.`